tutor/templates/config/base.yml

```diff
@@ -8,6 +8,8 @@ OPENEDX_SECRET_KEY: "{{ 24|random_string }}"
 PLUGINS:
   # The MFE plugin is required
   - mfe
+  # SurveyReport plugin is required
+  - surveyreport
```

Comment:
We can not allow the survey report to become a requirement for every tutor install. We have a proposal to make it part of tutor full, but first we need to finish up the plugin and make it completely optiona. Only then we can create a separate request to include it in https://github.com/overhangio/tutor/blob/master/requirements/plugins.txt